### ISSUE TITLE
关于Zero-shot 和 Finetune 模式下 Acc 计算问题 

### ISSUE THREAD

**Issue by lulu51230** [TsinghuaAI/CPM-1-Finetune#35]
背景：Zero-shot 模式在 distill 模型的ACC是50+，Finetune 模式在distill模型（没有任何train interation，直接evaluate）的ACC 是10+
提问：1. 请问 Zero-shot 和 Finetune 的具体计算ACC方式的不同在哪里？
           2. 查看了两种模式的代码，详情如下，对于这两种模式的ACC计算有点不懂。
           2.1 Fintune 模式的evaluate：output = model(***)，...，pred = torch.argmax(score)#score是output的转换。这里计算ACC的pred与output直接相关
           2.2 Zero-shot 模式的evaluate：output = model(***)，loss = mpu.vocab_parallel_cross_entropy(output.contiguous().float(), no_model_batch["labels"])，...，preds[sid].append((cid, loss))，... , 这里计算ACC的pred与loss相关

**Comment by zzy14** [comment]
主要区别在于类的数量不同，zero-shot setting下我们对类别进行了采样，所以任务会更简单，具体您可以参考[论文](https://arxiv.org/pdf/2012.00413.pdf)的3.1的第三段。
